src/api/search/.dockerignore

```diff
@@ -0,0 +1,6 @@
+.dockerignore
```

Comment:
I'm really curious about this .dockerignore

Reply:
I'm understanding the structure of this project more now. Thank you! :)

Reply:
Asking great questions is how you learn, keep it up! Code review is about fixing issues, but it's also about communicating how things work, and it's a perfect time to ask questions so that the community understands why a change is being made, and what it does.

Reply:
> do we have two? oh I see now.

Just for future reference, we don't want the dockerignore file to be included as well so we add it here too 👍🏾